addons/counterMortar/functions/fnc_selectResponse.sqf

```diff
@@ -0,0 +1,48 @@
+#include "script_component.hpp"
+/*
+    Author:
+        Bridg
+
+    Description:
+        selects response between air units, ground units and counter battery
+
+    Parameters:
+        0: _counterBatteryUnit <OBJECT>
+
+    Return value:
+        Nothing
+*/
+
+params ["_counterBatteryUnit"];
+
+if (count (GVAR(groundVehicleTypes) + GVAR(airVehicleTypes) + GVAR(counterBatteryUnits)) == 0) exitWith {diag_log "UKSF: ANTI MORTAR --- No counter units defined, check preInit ---"};
```

Comment:
I don't think the fired EH should even be added if there are no counter units defined